src/app/todolist/domain/todo.ts

```diff
@@ -1,5 +1,5 @@
 export class Todo {
-  id: number;
-  title: string;
-  done: boolean;
+  id? = 0;
```

Comment:
初期値入れるならnull許容(?)じゃなくてもよさそう

```suggestion
  id? = 0;
```

Reply:
> あとidは自動採番してるからid?: numberでnull許可してもいいかもね

titleは画面で、doneはreducerでそれぞれ自動で値が設定されるので全てnull許容にしてしまい、コンストラクタを廃止しました